libexec/apple-llvm/git-apple-llvm-mt-translate-branch

```diff
@@ -125,12 +134,46 @@ translate_branch() {
     fi
 
     list_new_commits "$branch" "${listdirs[*]}" "${repeated[*]}" $until |
+    process_new_commits
+    local status=$?
+    [ $status -ne $REPORT_WORK ] || exit 0
+    [ $status -eq 0 ] || exit $status
+}
+
+REPORT_WORK=27
```

Comment:
What does `27` mean here? Should it be commented/configurable?

Reply:
It was a tristate, but apparently it's not necessary at all.  If the caller needs to know whether work was reported, it can capture the `/dev/stdout` of the pipeline that `process_new_commits` is part of... but that's not even necessary here.  I'm not sure why I thought `translate_branch` needed to know this, but the tests pass without.  Removed in 5582cec2d3149e9a5bd10728da5009baae86888e.